File: CodeForces/101-200/155/A/155A.py
# Problem 155A: I_love_\%username\%
# https://codeforces.com/contest/155/problem/A
# Executed with: Python 3.11.2
# Linux version: Linux kernel 6.1.0-10-amd64
# Submitted on: September 26th, 2024
# Codeforces language used: PyPy 3.10 (7.3.15, 64bit)

input()
scores = list(map(int, input().split()))

high_score = scores[0]
low_score = scores[0]

ans = 0

for score in scores:
    if score > high_score:
        ans += 1
        high_score = score
    elif score < low_score:
        ans += 1
        low_score = score
print(ans)

File: CodeForces/101-200/155/A/155A.java
// Problem 155A: I_love_\%username\%
// https://codeforces.com/contest/155/problem/A
// Compiled with: openjdk 17.0.16 2025-07-15
// Executed with: javac 17.0.16
// Linux version: Linux kernel 6.1.0-10-amd64
// Submitted on: October 22nd, 2025
// Codeforces language used: Java 21 64bit

import java.util.Scanner;
 
public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();

        int lowest = sc.nextInt();
        int highest = lowest;

        int answer = 0;
        for (int i = 1; i < n; i++) {
            int nextScore = sc.nextInt();
            if (nextScore > lowest) {
                answer++;
                lowest = nextScore;
            } else if (nextScore < highest) {
                answer++;
                highest = nextScore;
            }
        }
        System.out.println(answer);
        sc.close();
    }
}
